Documentation~/meshes-as-assets.md

```diff
@@ -0,0 +1,11 @@
+# Store meshes as assets
+
+By default, ProBuilder stores its meshes in the Scene asset, so you can't access them as individual assets. If you need to access the meshes, for example to share them with other projects, you can store them as standalone assets in the Project folder.
+
+To save a ProBuilder mesh as a standalone asset:
```

Comment:
Maybe it would be more correct to say that "To enable ProBuilder to save meshes as assets by default..." or something similar along these lines as otherwise it sounds as if the act of selecing "Store mesh as asset" will perform an action/immediately save a selection/etc.